NetCasbin/InternalEnforcer.cs

```diff
@@ -387,6 +387,7 @@ private void NoticePolicyChanged(string section)
             if (autoBuildRoleLinks && section.Equals(PermConstants.Section.RoleSection))
             {
                 BuildRoleLinks();
+                ExpressionProvider.ClearFunctions();
```

Comment:
unnecessary.

Reply:
The cache of GFunction needs to refresh. If delete it the unit test can not pass.

Reply:
but it will clear user add function.

Reply:
You are right, I have recommitted to solve it.